libraries/chain/include/eosio/chain/block.hpp

```diff
@@ -93,9 +93,9 @@ namespace eosio { namespace chain {
    public:
       signed_block() = default;
       explicit signed_block( const signed_block_header& h ):signed_block_header(h){}
-      signed_block( signed_block&& ) = default;
+      signed_block( signed_block&& ) = default;                // can be =delete once http push_block removed
```

Comment:
remove comment, there is no reason to disable move constructor on `signed_block`

Reply:
Removed